diff --git a/modules/block_page_layout/src/Plugin/DisplayVariant/BlockPageLayoutVariant.php b/modules/block_page_layout/src/Plugin/DisplayVariant/BlockPageLayoutVariant.php
index d5a4bc8..22ee6de 100644
--- a/modules/block_page_layout/src/Plugin/DisplayVariant/BlockPageLayoutVariant.php
+++ b/modules/block_page_layout/src/Plugin/DisplayVariant/BlockPageLayoutVariant.php
@@ -75,9 +75,9 @@ class BlockPageLayoutVariant extends BlockDisplayVariant {
    * {@inheritdoc}
    */
   public function build() {
-    $regions = parent::build();
+    $build = parent::build();
     $layout = $this->getLayout();
-    $build['regions'] = $layout->build($regions['regions']);
+    $build['regions'] = $layout->build($build['regions']);
     return $build;
   }
 
